ios/brave-ios/Sources/BraveShared/Extensions/ProcessInfoExtensions.swift

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2024 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import Foundation
+import LocalAuthentication
+
+extension ProcessInfo {
+  public var isiOSAppOnVisionOS: Bool {
```

Comment:
Why not check UIDevice.current.userInterfaceIdiom == .vision

Reply:
Because it's not reported as `vision` because the app is running in compatibility mode, so it'll return `pad` always